repos/system_upgrade/el7toel8/files/rhel_upgrade.py

```diff
@@ -90,7 +90,10 @@ def run(self):
         self.base.distro_sync()
 
         if self.opts.tid[0] == 'check':
-            self.base.resolve(allow_erasing=self.cli.demands.allow_erasing)
+            try:
+                self.base.resolve(allow_erasing=self.cli.demands.allow_erasing)
+            except Exception as e:
```

Comment:
Could you use more specific exception(s)?

Reply:
ouch, I was almost sure I was opening this PR just against my own fork to show @vinzenz the diff. But anyway I will have to test it, so setting WIP...

Reply:
fixed